packages/augur-sdk/src/utils.ts

```diff
@@ -178,3 +209,51 @@ export function calculatePayoutNumeratorsArray(
   }
   return payoutNumerators;
 }
+
+export enum YesNoOutcomes {
```

Comment:
these should be in type file next to this:

```typescript
export enum MarketTypeName {
  YesNo = 'yesNo',
  Categorical = 'categorical',
  Scalar = 'scalar',
}
```

imo

Reply:
I like it. Moved!